packages/react/src/patterns/blocks/ContentBlockHeadlines/__stories__/ContentBlockHeadlines.stories.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { object, text } from '@storybook/addon-knobs';
+import ContentBlockHeadlines from '../ContentBlockHeadlines';
+import { DDS_CONTENTBLOCK_HEADLINES } from '../../../../internal/FeatureFlags';
+import items from './data/items.json';
```

Comment:
@kennylam we should probably get out of the habit of json blocks for knobs, any way to break this down?

Reply:
@jeffchew I've been of the opinion for a while that we shouldn't include those objects as knobs at all. Especially with our plans to create Codesandbox examples for everything, it's not needed. I'd say we just remove them entirely.

Reply:
As discussed, we should either have dynamic item knobs like http://ibmdotcom-react.mybluemix.net/?path=/story/components-buttongroup--default, or remove for now and we address later.

Reply:
@jeffchew Removing for now. Will revisit the story later and look into adding dynamic items that can have randomized content to test `sameHeight` and CTA or not.